src/components/StepOne/StepOne.css

```diff
@@ -1,16 +1,19 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,800;1,300&display=swap");
+
 .StepOne_btn {
   height: 1.6rem;
   width: 13.5rem;
-  background-color: red;
+  background-color: #da1b24;
```

Comment:
Always use css variables for color. It's difficult to main them like this. Refer to [this](https://github.com/RipeSeed/metaengine/blob/main/src/styles/helpers/_variables.scss) to see how we should handle variables. I know you are using css and the attached link is using scss but that would be a good starting place to look at.